content/blog/kubecon-na-2021/index.md

```diff
@@ -0,0 +1,40 @@
+---
+path: '/kubecon-na-2021'
+title: 'Come meet us at Kubecon NA 2021'
+date: 2021-10-12T00:00:00.000+00:00
+featured_image: hero-image.png
+categories: ['Pixie Team Blogs', 'Announcements', 'Kubernetes']
+authors: ['Zain Asgar', 'Michelle Nguyen']
+emails: ['zasgar@pixielabs.ai', 'michelle@pixielabs.ai']
+featured: true
+---
+Pixie will be attending this year's Kubecon + Cloud NativeCon NA in Los Angeles! Pixie was accepted to CNCF’s Sandbox in June 2021 and we have just officially completed the [Sandbox onboarding process](https://github.com/cncf/toc/issues/674). As a newly fledged CNCF Sandbox project, we're excited to connect with the community.
+
+Come chat observability, eBPF, open-source, and more with Pixie’s core contributors at our booth in the [Project Pavilion](https://events.linuxfoundation.org/wp-content/uploads/2021/10/KubeCon_NA_2021_Collateral_17x11_v2.pdf) from 10:30am-4:30pm, on all event days.
+
+We will also be holding three events this year:
+- [**Pixie Happy Hour**](https://pixiehh.eventbrite.com/) (Tuesday 10/12, 8:30pm @ Spire 73)\
+Discuss Pixie, eBPF, and Kubernetes over appetizers, drinks, and sweeping views of Los Angeles from Spire 73, the tallest open-air bar in the Western hemisphere.
+
```

Comment:
I don't think we actually need the newline between these points... Looks too spacious.

Reply:
agreed. will fix.